gammapy/maps/wcsnd.py

```diff
@@ -648,10 +651,19 @@ def cutout(self, position, width, mode='trim', copy=True):
         -------
         cutout : `~gammapy.maps.WcsNDMap`
             The cutout map itself
-        cutout_slices : Tuple of slice objects (with dimension 1 less than that of the non-spatial axes of the map)
         """
         idx = (0,) * len(self.geom.axes)
 
+        width = _check_width(width)
+
+        # We revert the order to comply with astropy.cutout2D ordering
+        if len(width) == 2:
```

Comment:
There should be no if / else needed here, `_check_width` should always return a length 2 tuple.

So remove this and below just pass this to cutout2d
```
size = width[::-1] * u.deg
```

Reply:
Done